src/map_query.rs

```diff
@@ -328,17 +328,91 @@ impl<'a> MapQuery<'a> {
         }
     }
 
-    /// Lets the internal systems know to "remesh" the chunk.
+    /// Retrieves a list of neighbor entities in the following order:
+    /// N, S, W, E, NW, NE, SW, SE.
+    ///
+    /// The returned neighbors are tuples that have an tilemap coordinate and an Option<Entity>.
+    ///
+    /// A value of None will be returned for tiles that don't exist.
+    ///
+    /// ## Example
+    ///
+    /// ```
+    /// let neighbors = map.get_tile_neighbors(UVec2::new(0, 0));
+    /// assert!(neighbors[1].1.is_none()); // Outside of tile bounds.
+    /// assert!(neighbors[0].1.is_none()); // Entity returned inside bounds.
+    /// ```
+    pub fn get_tile_neighbors<M: Into<u16>, L: Into<u16>>(
+        &self,
```

Comment:
This looks like a bad merge/rebase to me. Didn't this code move into a separate file?

Reply:
Yep. I'll go over it and check again.